sip-core/src/main/java/de/ikor/sip/foundation/core/declarative/DeclarationsRegistry.java

```diff
@@ -57,11 +60,111 @@ public DeclarationsRegistry(
 
     createMissingConnectorGroups();
     checkForDuplicateConnectorGroups();
+    checkForMissingConnectorGroupParent();
+    checkForUnusedMappers();
     checkForDuplicateScenarios();
+    checkForMissingScenarioParent();
+    checkForMissingConnectorParent();
     checkForUnusedScenarios();
     checkForDuplicateConnectors();
   }
 
+  private void checkForUnusedMappers() {
+    connectors.forEach(
+        connectorDefinition -> {
+          if (connectorDefinition instanceof ConnectorBase base) {
+            base.getRequestMapper()
+                .ifPresent(
+                    mapper -> {
+                      if (isRequestMappingOverridden(connectorDefinition, mapper)) {
+                        throw SIPFrameworkInitializationException.init(
+                            "Request mapping in connector '%s' is defined, but overridden",
```

Comment:
Leave a more descriptive message. It's overridden by what - I would guess the transformation? Leave instructions like either remove the mapping or request transformation defined in ...same applies for the other side (response mapper)